AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ReconcileAndLocalSaveOperation+Action.swift

```diff
@@ -23,7 +23,7 @@ extension ReconcileAndLocalSaveOperation {
 
         /// Operation has applied the incoming RemoteModel to the local database per the reconciled disposition. This
         /// could result in either a save to the local database, or a delete from the local database.
```

Comment:
What is the `Bool` member for?

Reply:
[here](https://github.com/aws-amplify/amplify-ios/pull/758/files#diff-fa89be203fdfd733316628da6f8384ffR260), I need to query `MutationSyncMetadata` locally to see if it is `create` type to local store if the version of coming event is not 1. The `Bool` here is to pass the `isCreated` value to `func notify()`